packages/client/src/client/App/Trades/GridRegion/components/ComparatorSelect.tsx

```diff
@@ -10,18 +10,26 @@ export const ComparatorSelectOuter = styled.div`
   display: block;
   position: relative;
   width: 10rem;
-  background-color: ${({ theme }) => theme.core.lightBackground};
-  color: ${({ theme }) => theme.core.textColor};
-  font-size: 12px;
+  background-color: ${({ theme }) =>
+    theme.newTheme.color["Colors/Background/bg-secondary"]};
+  color: ${({ theme }) =>
+    theme.newTheme.color["Colors/Text/text-primary (900)"]};
   outline: none;
   padding: 8px 15px 5px 10px;
   cursor: pointer;
   transition: all 200ms ease;
   text-transform: none;
   text-align: left;
-  border-bottom: 1px solid ${colors.spectrum.blue.base};
+  border-bottom: 1px solid
+    ${({ theme }) => theme.newTheme.color["Colors/Border/border-brand"]};
+
+  &:hover {
+    background-color: ${({ theme }) =>
+      theme.newTheme.color["Colors/Background/bg-secondary_hover"]};
+  }
 `
-const textColor = ({ theme }: { theme: Theme }) => theme.textColor
+const textColor = ({ theme }: { theme: Theme }) =>
+  theme.newTheme.color["Colors/Text/text-primary (900)"]
```

Comment:
aside: type within the styled comps template text is DefaultTheme .. would this look nicer with a named type?
(obvs can extract this to a more shareable spot if used elsewhere ...)
e.g.

```suggestion
import styled, { DefaultTheme } from "styled-components"

type ThemeFunction = (args: { theme: DefaultTheme }) => string

const textColor: ThemeFunction = ({ theme }) =>
  theme.newTheme.color["Colors/Text/text-primary (900)"]
```